C23_WMDE_Desktop_DE_14

Abban opened the pull request:
- Based on the CTRL of test 13.
- VAR banner contains a live date and time.

Ticket: https://phabricator.wikimedia.org/T351623

gbirke commented on shared/components/FundsModal.jsx:
```diff
@@ -0,0 +1,86 @@
+		super( props );
+		this.locale = props.locale;
+	}
+	formatAmount = function ( amount ) {
```
Maybe pass in numberFormatters (initialized from the `format-number` library) as properties and only do the-space-to-number conversion in the component.

gbirke commented on shared/components/FundsModal.jsx:
```diff
@@ -0,0 +1,86 @@
+			}
+			targetElement.classList.add( 'is-active' );
+		}
+	};
```
Ok for now, but to my knowledge it's not a good practice to manipulate the DOM that was rendered by React. Since we're using React as a templating engine, it's ok, but future versions of this component should use reactive sub-components.

Abban committed:
Add FallbackBannerConductor

This banner conductor will show a fallback banner if the
width is below a threshold or the main banner has a size
issue.

Ticket: https://phabricator.wikimedia.org/T351323

Abban committed:
Update name of Sofort test

This test checks both forms, not only inside the range
for upgrade to monthly

Abban committed:
Archive C23_WMDE_Desktop_DE_13

Ticket: https://phabricator.wikimedia.org/T351623

Abban committed:
Prepare campaign for C23_WMDE_Desktop_DE_14

Based on the CTRL of test 13.

Ticket: https://phabricator.wikimedia.org/T351623

Abban committed:
Implement VAR for C23_WMDE_Desktop_DE_14

VAR banner contains a live date and time.

Ticket: https://phabricator.wikimedia.org/T351623

Abban committed:
Hook up Fallback Banner
